modules/oe_webtools_laco_service/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenEuropa Webtools Laco Service
+
+A Webtools Laco service that provides information on whether entity has a translation in a specific language.
+
+### How it works
+
+This module exposes a new route for each entity type that "mirrors" the canonical route of entities. These routes have 
+the same paths as the canonical entity routes but use different requirements for matching. If these requirements
+are matched in the request, the module will return Laco information for that entity.
+
+### How to test
```

Comment:
I'm not sure if we need to add this section to the Readme but I won't block the PR because of it.